common/djangoapps/course_modes/models.py

```diff
@@ -94,8 +94,8 @@ class CourseMode(models.Model):
     NO_ID_PROFESSIONAL_MODE = "no-id-professional"
     CREDIT_MODE = "credit"
 
-    DEFAULT_MODE = Mode(HONOR, _('Honor Code Certificate'), 0, '', 'usd', None, None, None)
-    DEFAULT_MODE_SLUG = HONOR
+    DEFAULT_MODE = Mode(AUDIT, _('Audit'), 0, '', 'usd', None, None, None)
+    DEFAULT_MODE_SLUG = AUDIT
```

Comment:
@peter-fogg I know this is a WIP, but would it make sense to use this change as an opportunity to get away from the notion of a "default mode" altogether? The problems we've encountered in the past have been due to the fact that the LMS assumes a default enrollment mode, not necessarily that the default mode happens to be honor.